uberduck_ml_dev/exec/utils.py

```diff
@@ -0,0 +1,14 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/exec.utils.ipynb (unless otherwise specified).
+
+__all__ = ['parse_args']
+
+# Cell
+import argparse
+import sys
+
+# Cell
+def parse_args(args):
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--config", help="Path to JSON config")
+    args = parser.parse_args(args)
+    return args
```

Comment:
Can you move this file to `utils/exec.py` o `utils/argparse.py`? `exec` should be only scripts with a `__main__` block, imports should come from elsewhere.